django/db/models/fields/__init__.py

```diff
@@ -630,6 +630,12 @@ def db_type(self, connection):
         except KeyError:
             return None
 
+    def rel_db_type(self, connection):
+        """
+        Returns the database column data type for related field referencing to this.
```

Comment:
for new code we're using PEP257 verb style "Return..."
"for related field referencing to this" isn't grammatically correct.